Podcast/NullProfileCollectionViewCell.swift

```diff
@@ -7,24 +7,57 @@
 //
 
 import UIKit
+import SnapKit
 
 class NullProfileCollectionViewCell: UICollectionViewCell {
+    //combine with external profile null cell
+    //conditinoally define height for if current user or not
+    let addIconSize: CGFloat = 16
     
-    let addIconSize = 16
-    
+    //current user null profile
     var addIcon: UIImageView!
     
+    //other user null profile
+    var nullLabel: UILabel!
+    let labelHeight: CGFloat = 21
+    
+    var cellHeight: CGFloat = 100
+    
     override init(frame: CGRect) {
```

Comment:
since these don't have reuuse identifiers accourding to my comment below and we really need a user to intilize it then make the init(user: User)

Reply:
Same with the other cell